modules/mobile/app/serializers/mobile/v0/facilities_info_serializer.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class FacilitiesInfoSerializer
+      include FastJsonapi::ObjectSerializer
+
+      set_type :facility_info
+      attributes :facilities
+
+      def initialize(id, facilities)
+        resource = FacilityInfoStruct.new(id, facilities)
```

Comment:
I've seen this done a couple ways in other mobile endpoints. Here the array of objects is within the attributes. sometimes our endpoints have the array be the `data` field. The Json API spec says it should be the latter but that can lead to some unintuitive formatting, especially when we don't use a model. So I'm hesitant to say we should blindly follow the spec if it doesn't serve us well. This may be a good place to discuss it though so we're consistent.

Reply:
Yeah, I just assumed always having the pertinent response info inside attributes was a little more intuitive and see we've done it like that elsewhere I did it again here. Overall I don't have much of a preference though.